lib/std/debug.zig

```diff
@@ -846,6 +846,7 @@ fn readCoffDebugInfo(allocator: mem.Allocator, coff_file: File) !ModuleDebugInfo
                 .debug_info = debug_info_data orelse return error.MissingDebugInfo,
                 .debug_abbrev = debug_abbrev_data orelse return error.MissingDebugInfo,
                 .debug_str = debug_str_data orelse return error.MissingDebugInfo,
+                .debug_str_offsets = null,
```

Comment:
Would it be possible for you to add COFF support for this segment, or else a more specific error or a useful panic?

I'd like to avoid reporting `error.InvalidDebugInfo` when the real problem is that the standard library support is incomplete - Makes it much quicker to root cause 🙂

Reply:
The thing is I don't have a windows machine and I don't want to risk introducing a regression. As you know this code is only triggered when a compiled executable hits an error, so I could imagine errors introduced here not getting caught by CI. I don't know, but I figured I'd play it safe if I'm not regressing anything my omitting a feature / change.

I was also just trying to keep this PR simple and not get into refactoring as it's my first, I can see that there's a lot of simple improvements that could be made in this file.

I think the best place for this error would be in the `getAttrString` function. If a string of form type `StrOffset` is requested but we haven't loaded a `debug_str_offsets` table then we can report that to the user. Does that sound reasonable?

I will check the specification in a sec, but I'm not 100% sure that `debug_str_offsets` is a mandatory section in V5 if there isn't any string data of this form type.

Reply:
That error location sounds good to me :+1: